src/config.json

```diff
@@ -19,6 +19,32 @@
     "launchpad.ethereum.org"
   ],
   "whitelist": [
+    "opengear.com",
+    "opinsta.com",
+    "opensc.org",
+    "affinity.locus",
+    "opendev.org",
+    "opensee.io",
+    "openlca.org",
```

Comment:
This domain isn't mentioned in any issues :thinking:

Reply:
I report 13 days ago 
https://github.com/MetaMask/eth-phishing-detect/issues/5574

Domain punctus.org
Hope you can whitelist this domain, thanks!

Reply:
@radarzeo please look at line # 41